app/context/userContext.tsx

```diff
@@ -168,6 +175,7 @@ export const UserContextProvider = ({ children }: { children: any }) => {
         );
         setCeramicDatabase(ceramicDatabaseInstance);
         fetchPassport(ceramicDatabaseInstance);
+        setUserDid(viewerConnection.selfID.id);
```

Comment:
The `selfID.id` is now a session DID (changes each time we authenticate) - is that what you want, or are we after the `did:pkh`?